### ISSUE TITLE
If "forceConsistentCasingInFileNames": true in `tsconfig.json`, an error occurs.

### ISSUE THREAD

**Issue by yaegassy** [Stillat/blade-parser-typescript#4]
## Description

It seems that the following is the problem.

https://github.com/Stillat/blade-parser-typescript/blob/0073e2db43a38eb0dd490325aa96f73e704bb740/src/document/documentCursor.ts#L2

I think the issue can be resolved by changing it to `import { BladeDocument } from './bladeDocument';`.

### 1. Prepare

```bash
mkdir -p /tmp/blade-parser-check
cd /tmp/blade-parser-check
npm init -y
npm i stillat-blade-parser
npm i -D typescript
npx tsc --init
touch sample.ts
```

---

The following is a modified version of `tsconfig.json` generated by `tsc --init`, with comments removed. 

In `tsconfig.json`, change `skipLibCheck: false`.

**tsconfig.json**:

```jsonc
{
  "compilerOptions": {
    "target": "es2016",                                  
    "module": "commonjs",                               
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true, // <-
    "strict": true,                         
    "skipLibCheck": false // <-
  }
}
```

## 2. Code

The following is an example code of using a simple parser described in the README.

**sample.ts**:

```typescript
import { BladeDocument } from "stillat-blade-parser/out/document/bladeDocument";

const document = BladeDocument.fromText(`<html>
    <head>
        <title>{{ $title }}</title>
    </head>
</html>`);
```

### 3. Run npx tsc --noEmit

Unfortunately, an error occurs...

```bash
$ npx tsc --noEmit
node_modules/stillat-blade-parser/out/document/documentCursor.d.ts:2:31 - error TS1149: File name '/private/tmp/blade-parser-check/node_modules/stillat-blade-parser/out/document/BladeDocument.d.ts' differs from already included file name '/private/tmp/blade-parser-check/node_modules/stillat-blade-parser/out/document/bladeDocument.d.ts' only in casing.
  The file is in the program because:
    Imported via "stillat-blade-parser/out/document/bladeDocument" from file '/private/tmp/blade-parser-check/sample.ts' with packageId 'stillat-blade-parser/out/document/bladeDocument.d.ts@0.1.3'
    Imported via '../document/bladeDocument' from file '/private/tmp/blade-parser-check/node_modules/stillat-blade-parser/out/nodes/nodes.d.ts' with packageId 'stillat-blade-parser/out/document/bladeDocument.d.ts@0.1.3'
    Imported via './BladeDocument' from file '/private/tmp/blade-parser-check/node_modules/stillat-blade-parser/out/document/documentCursor.d.ts' with packageId 'stillat-blade-parser/out/document/BladeDocument.d.ts@0.1.3'
    Imported via './bladeDocument' from file '/private/tmp/blade-parser-check/node_modules/stillat-blade-parser/out/document/documentErrors.d.ts' with packageId 'stillat-blade-parser/out/document/bladeDocument.d.ts@0.1.3'
    Imported via '../bladeDocument' from file '/private/tmp/blade-parser-check/node_modules/stillat-blade-parser/out/document/scanners/nodeScanner.d.ts' with packageId 'stillat-blade-parser/out/document/bladeDocument.d.ts@0.1.3'
    Imported via './bladeDocument' from file '/private/tmp/blade-parser-check/node_modules/stillat-blade-parser/out/document/transformer.d.ts' with packageId 'stillat-blade-parser/out/document/bladeDocument.d.ts@0.1.3'
    Imported via './bladeDocument' from file '/private/tmp/blade-parser-check/node_modules/stillat-blade-parser/out/document/formatters.d.ts' with packageId 'stillat-blade-parser/out/document/bladeDocument.d.ts@0.1.3'

2 import { BladeDocument } from './BladeDocument';
                                ~~~~~~~~~~~~~~~~~

  sample.ts:1:31
    1 import { BladeDocument } from "stillat-blade-parser/out/document/bladeDocument";
                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    File is included via import here.
  node_modules/stillat-blade-parser/out/nodes/nodes.d.ts:3:31
    3 import { BladeDocument } from '../document/bladeDocument';
                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    File is included via import here.
  node_modules/stillat-blade-parser/out/document/documentErrors.d.ts:2:31
    2 import { BladeDocument } from './bladeDocument';
                                    ~~~~~~~~~~~~~~~~~
    File is included via import here.
  node_modules/stillat-blade-parser/out/document/scanners/nodeScanner.d.ts:3:31
    3 import { BladeDocument } from '../bladeDocument';
                                    ~~~~~~~~~~~~~~~~~~
    File is included via import here.
  node_modules/stillat-blade-parser/out/document/transformer.d.ts:1:31
    1 import { BladeDocument } from './bladeDocument';
                                    ~~~~~~~~~~~~~~~~~
    File is included via import here.
  node_modules/stillat-blade-parser/out/document/formatters.d.ts:1:31
    1 import { BladeDocument } from './bladeDocument';
                                    ~~~~~~~~~~~~~~~~~
    File is included via import here.


Found 1 error in node_modules/stillat-blade-parser/out/document/documentCursor.d.ts:2
```

**Comment by JohnathonKoster** [comment]
Thanks for the report!

I've made the change, and published a new version on NPM (0.1.4)

**Comment by yaegassy** [comment]
Thanks 🙇
